microproxy/event/manager.py

```diff
@@ -10,11 +10,13 @@
 
 
 class EventManager(object):
-    def __init__(self, config, proxy_server, handler=None, zmq_stream=None):
+    def __init__(self, config, handler=None, zmq_stream=None):
         super(EventManager, self).__init__()
-        self.handler = handler or EventHandler(config, proxy_server)
+        self.handler = handler or EventHandler(config)
         self.zmq_stream = zmq_stream or self._create_stream(config)
-        self._start()
+
+    def start(self):
+        self.zmq_stream.on_recv(self._on_recv)
```

Comment:
Nice work on it for start the manager with explicit call that make it more clearly!